tests/testdata/ecdsa.key

```diff
@@ -1,5 +1,5 @@
 -----BEGIN EC PRIVATE KEY-----
-MHcCAQEEINL8ATxoL9YROkZ0KN4psqHHkfuXS7CnoKHTpQcJ1rW7oAoGCCqGSM49
-AwEHoUQDQgAE/zrRppSJW7b7ydzj9HZuZchDAqzJ6Q0w48AT51Gdn/teiwpjbHbR
-CdK9EneBjKOC7LNiacMz8tkixhI9ysBikg==
+MHcCAQEEINHLf686Kw39K94sKXC+oYNs45qVfnXbUBacgRDuCOHQoAoGCCqGSM49
```

Comment:
I'm just curious, why did these certs get changed?

Reply:
The previous key/certs would not work with SoftHSM2!

Reply:
Why not?

Reply:
I don't know. Try it!

Reply:
Sounds like a bug, or at least non-linear behavior that will trip us up in the future. I'd recommend looking into it.

Reply:
@mahrud What was the issue here?

Reply:
I'm not sure and I can't replicate it anymore. It might have been me doing something incorrectly.
I've reverted the changes though, the keys haven't changed. I just moved them from `pkg` into `tests/testdata` to avoid installing testing keys.